mcs/class/System.ServiceModel/System.ServiceModel.Description/MetadataExchangeClient.cs

```diff
@@ -88,7 +89,17 @@ public MetadataExchangeClient (Uri address, MetadataExchangeClientMode mode)
 		[MonoTODO]
 		public int MaximumResolvedReferences { get; set; }
 
-		public TimeSpan OperationTimeout { get; set; }
+		public TimeSpan OperationTimeout {
+			get {
+				if (!this.operation_timeout.HasValue) {
+					this.operation_timeout = TimeSpan.FromMinutes (1);
```

Comment:
No idea why it's the default, but it was set elsewhere. I should move it out into a #define.